acceptance/builder_test.go

```diff
@@ -0,0 +1,184 @@
+package acceptance
+
+import (
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/lifecycle/api"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+)
+
+var (
+	builderDockerContext = filepath.Join("testdata", "builder")
+	builderBinaryDir     = filepath.Join("testdata", "builder", "container", "cnb", "lifecycle")
+	builderImage         = "lifecycle/acceptance/builder"
+	builderUserID        = "1234"
+)
+
+func TestBuilder(t *testing.T) {
+	h.SkipIf(t, runtime.GOOS == "windows", "builder acceptance tests are not yet supported on Windows")
+
+	rand.Seed(time.Now().UTC().UnixNano())
+
+	//h.MakeAndCopyLifecycle(t, "linux", builderBinaryDir)
```

Comment:
I suspect this is commented out for speed of development, since building the binaries takes time. Our Makefile is not supposed to recompile the binaries if nothing has changed, but I think something is wrong with it... when you run `make build-linux` on the command line you'll see that the second invocation is a no-op, but that's not true when exec'ing from the acceptance tests. I'm not sure what the issue is...

Reply:
Yeah I'm not changing anything related to lifecycle directly so for testing i commented out this line. But i should not commit...  Normally I would leave this in my local but I committed it because it escaped my notice. Let add `FIXME` comment this line too until end of development. I think in DRAFT PR request there is no CI-CD for tests. So lets skip it until this pr get out of the draft. But i am adding `FIXME`comment anyway.